dockermake-cross.sh

```
@@ -7,4 +7,4 @@ ROOT=$(pwd)
 cd $CWD
 
 docker pull ghcr.io/grumpycoders/pcsx-redux-build-cross:latest
-docker run --rm --env-file ${ROOT}/cross-env.list -i -w/project${CWD#$ROOT} -v "${ROOT}:/project" -u `id -u`:`id -g` ghcr.io/grumpycoders/pcsx-redux-build-cross make CROSS=arm64 $@
+docker run --rm --env-file ${ROOT}/cross-env.list -i -w/project${CWD#$ROOT} -v "${ROOT}:/project" -u `id -u`:`id -g` --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined ghcr.io/grumpycoders/pcsx-redux-build-cross make CROSS=arm64 $@
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)